pymonetdb/mapi.py

```diff
@@ -101,6 +102,38 @@ def connect(self, database, username, password, language, hostname=None,
         unix_socket is used if hostname is not defined.
         """
 
+        if ':' in database:
```

Comment:
maybe we should move the param parsing to another function, but that is not really blocking for now.